care/facility/summarisation/patient_summary.py

```diff
@@ -0,0 +1,112 @@
+from datetime import timedelta
+
+from django.db.models import Q,Subquery
+from django.utils.timezone import now
+
+from celery.decorators import periodic_task
+from celery.schedules import crontab
```

Comment:
Codacy found an issue: [Unable to import 'celery.schedules'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5789522)